File: fib.js
function fib(n) {
  if (n < 2) {
    return 1;
  } else {
    return fib(n - 2) + fib(n - 1);
  }
}

module.exports = function(req, res) {
	var m = req.path.replace('/','');
	var n = parseInt(m);
	if( isNaN(n) )
		res.end();
	else{
		console.warn(n);
		var startTime = (new Date())/1000;
	    var number=fib(n);
	    var time = (new Date())/1000 - startTime;
	    res.end(''+time);
	}
}

File: benchmark/fib/fib.java
public class fib {
    public static void main(final String[] args) {
        // System.out.println("指定された引数は、" + args.length + "個です");
        int n =1;
        try{
        	n = Integer.parseInt(args[0]);
        }catch(Exception e){
			System.out.println("aaaa" + n);
        }
        int fib = fib(n);
		System.out.println(fib);
    }
    public static int fib(int n) {
		if (n < 2) {
			return 1;
		} else {
			return fib(n - 2) + fib(n - 1);
		}
	}
}